From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Eric Biggers <ebiggers@google.com>
Date: Wed, 12 Aug 2020 17:15:48 -0700
Subject: _____ANNOTATION: remove ANNOTATIONS file_____

The ANNOTATION commits had to update this file as a workaround for
'git am' not supporting applying empty patches.

Signed-off-by: Eric Biggers <ebiggers@google.com>
---
 ANNOTATIONS | 32 --------------------------------
 1 file changed, 32 deletions(-)
 delete mode 100644 ANNOTATIONS

diff --git a/ANNOTATIONS b/ANNOTATIONS
deleted file mode 100644
--- a/ANNOTATIONS
+++ /dev/null
@@ -1,32 +0,0 @@
-pending upstream
-binder changes
-Android vendor hooks
-kbuild changes
-x86 changes
-arm and arm64 changes
-core kernel changes
-power management changes
-scheduler changes
-uid_sys_stats support
-cpufreq changes
-mm (memory management) changes
-security changes
-dma-buf changes
-driver framework and misc driver changes
-tty changes
-USB changes
-graphics changes
-sound changes
-media changes
-input and HID changes
-UFS driver changes
-MMC driver changes
-block changes
-device-mapper changes
-filesystem changes
-Incremental filesystem
-networking changes
-hardware-wrapped key support
-db845c (DragonBoard 845c) support
-Android kconfigs and build scripts
-Bringup patches
